pkgs/applications/editors/hmm/default.nix

```
@@ -0,0 +1,40 @@
+{ stdenv
+, lib
+, makeWrapper
+, php
+, ncurses
+, coreutils
+, xclip
+}:
+
+stdenv.mkDerivation rec {
+  name = "h-m-m";
+
+  src = builtins.fetchGit {
+    url = "https://github.com/nadrad/h-m-m.git";
+    rev = "ecf74433844e5542f78f64f8a79807818743adff";
+  };
```

Looking at the ofborg log https://gist.github.com/GrahamcOfBorg/ed85746086c9a10f1b5e8cc45d350e36
> nix-env failed:
error: access to URI 'https://github.com/nadrad/h-m-m.git' is forbidden in restricted mode
>       … while evaluating the attribute 'src' of the derivation 'h-m-m'

Is there a reason to use fetchGit here instead of 
```suggestion
  src = fetchFromGitHub {
    owner = "nadrad";
    repo = "h-m-m";
    rev = "ecf74433844e5542f78f64f8a79807818743adff";
    sha256 = "sha256-xxxxvT271ASEk1MFZQ/TPdIsDxbGZUCkde4nmKpVQCU=";
  };
```
Here, you would have to change the SHA-256, but it should show what to replace it with when you try to install this package and it fails.